src/base/QXmppTrustMessageKeyOwner.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2008-2021 The QXmpp developers
+ *
+ * Author:
+ *  Melvin Keskin
+ *
+ * Source:
+ *  https://github.com/qxmpp-project/qxmpp
+ *
+ * This file is a part of QXmpp library.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ */
+
+#include "QXmppTrustMessageKeyOwner.h"
+
+#include <QDomElement>
+
+#include "QXmppConstants_p.h"
+#include "QXmppUtils.h"
+
+///
+/// \class QXmppTrustMessageKeyOwner
+///
+/// \brief The QXmppTrustMessageKeyOwner class represents a key owner of the
+/// trust message as defined by \xep{0434} Trust Messages (TM).
+///
+/// \since QXmpp 1.5
+///
+
+class QXmppTrustMessageKeyOwnerPrivate : public QSharedData
+{
+public:
+    QString jid;
+    QList<QString> identifiersOfTrustedKeys;
+    QList<QString> identifiersOfDistrustedKeys;
```

Comment:
should better be `trustedKeysIds`

Reply:
I'm also not happy with the new variant, please don't use `of` in variable names, it's possible to avoid this.